db/CassSeeder

```diff
@@ -0,0 +1,26 @@
+//creating a keyspace
+CREATE KEYSPACE IF NOT EXISTS languagefeatures WITH replication = {'class' : 'SimpleStrategy', 'replication_factor' : 1};
+
+//creating a table
+CREATE TABLE languagefeatures.languages (
```

Comment:
this could be a node script eg. client.execute(create keyspace etc...)
.then(()=>{
return client.execute(create table etc...)
})
.then(() => {
return client.execute(copy etc)
})
.then(() => {
client.execute(copy etc...)
})